ARExperiment/README.md

```diff
@@ -8,6 +8,19 @@ This is a simple demo app that pairs with our investigation into ARKIT
 * `RecognizeObjectsViewController` Adds a 3D model to the scene using CoreML to recognise where to place the model. You can read more on the Machine Learning part in the [Making AR more precise with CoreML](https://www.novoda.com/blog/arkit-coreml/)
 * `LightsAnimationsViewController` Adds one 3D model to the scene using `ARAnchors` and allowing for animations, lights and shadows to be imported from the model
 
+## FileNames vs NodeNames
+
+At the beginning of every controller you will see there are around 3 or more string variables hardcoded
+  - `fileName` if you are assets are not on the route of `art.scnassets` folder you should add the folder name before the file name like so: `Banana/banana`
```

Comment:
I think this could be rewritten. Maybe something like 

> `fileName` is just the name of the model. If your assets are not directly under `art.scnassets` you should add the folder name before the file name like so: `Banana/banana`